docs/topics/mpp/mpp-connect-to-apis.md

```diff
@@ -72,7 +72,23 @@ internal actual fun writeLogMessage(message: String, logLevel: LogLevel) {
 The main rules regarding expected and actual declarations are:
 * An expected declaration is marked with the `expect` keyword; the actual declaration is marked with the `actual` keyword.
 * `expect` and `actual` declarations have the same name and are located in the same package (have the same fully qualified name).
-* `expect` declarations never contain any implementation code.
+* `expect` declarations never contain any implementation code and are abstract by default.
+
+> In interfaces, functions in actual declarations can be non-abstract and have a
+> body. To indicate that in `expect` declarations, explicitly mark functions as `open`:
+>
+> ```kotlin
+> expect interface Mascot {
+>     open fun name()
```

Comment:
This is certainly minor, but I guess it a little bit contradicts the general sense of Kotlin language: `name` sounds as a property for me. One could rewrite it as `getName` to make it sound more like a function, but then we explicitly recommend to not do that and use properties instead.

The reason I'm bringing this up is that sometimes some users are very nitpicky about examples we use in our public content, so it's not pointless to be extra-careful here